src/routes/Welcome.tsx

```diff
@@ -179,14 +211,13 @@ export function Welcome(props: IProps) {
         newsPreference: false,
       },
     } as any;
-    await memory.update((t: TransformBuilder) =>
-      AddRecord(t, userRec, user, memory)
-    );
+    await AddUserLocalOnly(userRec);
     await offlineSetup();
     return userRec.id;
   };
+
   const handleQuickOffline = () => {
-    localStorage.setItem('autoaddProject', 'true');
+    if (!hasOfflineUsers) localStorage.setItem('autoaddProject', 'true');
```

Comment:
why aren't we checking offline project here instead of users?

Reply:
On this one, the issue is that if you have an offline user, you have likely already created an offline project so you don't automatically want to create a new one each time you come back to the welcome screen and choose this option.

Reply:
I think checking offline projects might do the same thing and may help us with the case where the user doesn't actually create a project the first time they click this option...

Reply:
but shouldn't it check online projects then for go online?